lldb/include/lldb/API/SBDefines.h

```diff
@@ -136,6 +136,8 @@ class LLDB_API SBWatchpoint;
 class LLDB_API SBWatchpointOptions;
 class LLDB_API SBUnixSignals;
 
+typedef SourceLanguageName SBSourceLanguageName;
```

Comment:
I'm not sure a typedef is enough here. It ends up being little more than syntactic sugar, the final symbol will still contain `SourceLanguageName` instead of `SBSourceLanguageName`. To verify this, you can perform a build with this patch and search for `SBExpressionOptions::SetLanguage` and make sure it still retains the `SBSourceLanguageName` symbol.